server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -165,15 +161,16 @@ private boolean isBlockEligible(Block block) {
   @Override
   public ImmutableList<Block> getAllActiveBlocks() {
     if (allActiveBlockList == null) {
-      allActiveBlockList = getBlocks(this::showBlock, /* includeHidden= */ false);
+      allActiveBlockList = getBlocks((block) -> showBlock(block));
     }
     return allActiveBlockList;
   }
 
   @Override
   public ImmutableList<Block> getAllHiddenBlocks() {
     if (allHiddenBlockList == null) {
-      allHiddenBlockList = getBlocks(this::showBlock, /* includeHidden= */ true);
+      //
```

Comment:
nit (blocking): can we remove this commented line?

Reply:
yea for sure. deleted.